app/services/validation_beta_service.rb

```diff
@@ -172,6 +172,67 @@ def sit_with_unvalidated_participants_reminders
     end
   end
 
+  def sit_fip_with_unvalidated_participants_deadline_reminders
+    campaign = :unvalidated_participants_deadline_reminder
+
+    sign_in_url = Rails.application.routes.url_helpers.new_user_session_url(
+      host: Rails.application.config.domain,
+      **UTMService.email(campaign, campaign),
+    )
+
+    participant_start_url = Rails.application.routes.url_helpers.participants_start_registrations_url(
+      host: Rails.application.config.domain,
+      **UTMService.email(campaign, campaign),
+    )
+
+    InductionCoordinatorProfile
+      .joins(schools: %i[active_ecf_participant_profiles partnerships])
+      .includes(schools: { active_ecf_participant_profiles: %i[ecf_participant_eligibility ecf_participant_validation_data] })
+      .where(
+        school_cohorts: {
+          cohort_id: Cohort.current.id,
+          induction_programme_choice: "full_induction_programme",
+        },
+        ecf_participant_eligibility: {
+          participant_profile_id: nil,
+        },
+        ecf_participant_validation_data: {
+          participant_profile_id: nil,
+        },
+      )
+      .merge(Partnership.active)
+      .distinct
+      .find_each do |sit|
+        sit.schools
+          .joins(:school_cohorts, :partnerships)
+          .where(school_cohorts: { cohort_id: Cohort.current.id })
+          .merge(Partnership.active)
+          .find_each do |school|
+          participants = school
+            .school_cohorts
+            .find_by(cohort_id: Cohort.current.id)
```

Comment:
```suggestion
            .find_by(cohort_id: Cohort.current.id)
```

Should we respect `opt_out_of_updates` here or is that covered elsewhere?

Reply:
I guess we should check for that, good spot (there are 39 FIP school cohorts that have opted out)

Reply:
Ooh interesting, how did that happen?

Reply:
I think it's a bug, let's not exclude them. The one I had looked at transistioned from DIY and didn't get opted in

Reply:
Shall I take that back out again or shall we update the school cohorts in question?